ooniprobe/View/Settings/CustomURLViewController.m

```diff
@@ -44,8 +44,9 @@ -(IBAction)addRow:(id)sender{
 -(IBAction)run:(id)sender{
     urlArray = [[NSMutableArray alloc] init];
     for (NSString *url in self.urlsList){
-        if (![url isEqualToString:@"http://"] && ![url isEqualToString:@"https://"]){
-            Url *currentUrl = [Url checkExistingUrl:url];
+        NSString *sanitizedUrl= [url stringByTrimmingCharactersInSet:[NSCharacterSet newlineCharacterSet]];
+        if (![sanitizedUrl isEqualToString:@"http://"] && ![sanitizedUrl isEqualToString:@"https://"]){
```

Comment:
This is slightly different from the Android algorithm. Maybe we should use the same algorithm in both cases? What is the reason why here you've introduced this extra check?

Reply:
On iOS we have this nice `stringByTrimmingCharactersInSet` that takes a `NSCharacterSet` but on Android we are forced to use a regExp (that I don't personally like)

Reply:
The question is why here you check for `http://` and `https://` and on Android you don't

Reply:
On Android we use `Patterns.WEB_URL.matcher(value).matches()` that I believe does the same thing

Reply:
you're right, thanks!